share/spice/crackhash/crackhash.js

```diff
@@ -0,0 +1,36 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_crackhash = function(api_result){
+
+	if(!api_result || !api_result.hits || api_result.hits.length === 0) {
+	    return Spice.failed('Crackhash');
+	}
+        Spice.add({
+            id: "crackhash",
+
+            name: "crackhash",
+            data: api_result.hits,
+            meta: {
+                sourceName: "crackhash",
+                sourceUrl: sourceUrl,
+	        itemType: (api_result.hits.length === 1) ? 'crack hash' : 'crack',
+   	        searchTerm: decodeURIComponent(query)
+            },
+            normalize: function(item) {
+                return {
+                    title: api_result.title,
+		    url:(item.url) ? item.url : 'http://api.md5crack.com/crack/ZFCQCcpS1kLgbCQZ/' + item.objectID,
+                    subtitle: api_result.subtitle,
+                    image: api_result.icon
+                };
+            },
+            templates: {
+                group: 'text',
+                options: {
+                    content: Spice.crackhash.content,
```

Comment:
you don’t need this because you're supplying a `title`, `subtile` etc. `content` is meant for custom markup inside when needed.